pipeline-templates/job/code-build.yml

```diff
@@ -1,47 +1,47 @@
 parameters:
   SolutionBaseName:
-  BuildConfiguration:
 
 jobs:
 - job: CodeBuild
+  pool:
+    name: DAS - Continuous Integration

```

Comment:
This should use the new ubuntu 20 agents

Reply:
I've now changed the pool being used and added the demand to specify Ubuntu 20